_posts/2018-07-31-tutorial.md

```diff
@@ -0,0 +1,1274 @@
+---
+layout: page
+title:  "Tutorial"
+permalink: tutorial
+---
+
+This page is a summary of a Protelis Tutorial lectured by Danilo Pianini at the
+11th IEEE International Conference on Self-Adaptive and Self-Organizing Systems
+at the University of Arizona, in Tucson, on September 2017.
+
+*[Original slides](https://www.slideshare.net/DanySK/practical-aggregate-programming-with-protelis-saso2017)*
+
+*[Tutorial in the saso proceedings](https://doi.org/10.1109/FAS-W.2017.186)*
+
+## Workbench setup
+
+This tutorial leverages [the Alchemist simulator](http://alchemistsimulator.github.io/)
+for executing Protelis programs in a simulated network of devices.
+Details on the simulator are out of the scope of this tutorial, in this tutorial
+we will only introduce the elements strictly necessary for configurating the
+device network.
+
+### Prerequisites
+
+* Internet connection
+* A working installation of Java 8+
+* A terminal emulator
+  * Unix-like terminals are ok
+  * Should work also with the Windows cmd.exe
+    * Don’t ignore case sensitivity
+  * git (preferred), or a program to decompress a zip file
+
+### Download
+
+* In the following, `WORKSPACE` represents the folder you want to work into
+
+* **Option 1** - git is installed on the system. Issue:
+  * ```bash
```

Comment:
These aren't rendering correctly on GitHub's .md viewer --- do they work on the webpage?

Reply:
A preview of the page is available [here](https://candogan96.github.io/Protelis-CD.github.io/tutorial)